docs/Building-Transmission.md

```diff
@@ -120,6 +120,90 @@ $ sudo make install
 ```
 
 ## On Windows ##
+
+## Building transmission-daemon
+You need the following installed:
+
+* Visual Studio 2017 (the Community Edition is sufficient - just make sure its C[++] compiler, MSVC, is installed)
```

Comment:
We're not testing with MSVS 2017, not sure if it's a good idea to use it in the documentation. Binaries for 4.0.0-beta.1 were prepared with MSVS 2019, 4.0.0-beta.2 will be using MSVS 2022. IIUC all versions include a free Community edition, and a free Build Tools package which is also sufficient.

Reply:
But i only have 2017 with Pro license.

Reply:
Then use 2022 Community Edition...